src/Microsoft.SqlTools.ServiceLayer/Scripting/ScriptingScriptOperation.cs

```diff
@@ -218,16 +217,52 @@ private SqlScriptPublishModel BuildPublishModel()
                     "Scripting object count {0}, objects: {1}",
                     selectedObjects.Count(),
                     string.Join(", ", selectedObjects)));
-
+            
+            // Get the server name explicitly after connecting to the server.
+            // This is required to catch the actual server name when targeting sql server on docker in linux.
```

Comment:
Can you restore the comment?

---
In reply to: [127087410](https://github.com/Microsoft/sqltoolsservice/pull/410#discussion_r127087410) [](ancestors = 127087410,127084230)